app/views/calculators/calculator.html.slim

```diff
@@ -25,12 +25,14 @@
             - (0..6).each do |month|
               option value = month
                 = month
-          p.form-a
-            = t '.form_price'
-            select.custom-select#childs_months [required]
-              - ([(t('.form.budgetary')), (t('.form.medium')), (t('.form.premium'))]).each do |price|
-                option value = price
-                  = price
+          - if FeatureFlag.get('feature_budget_category').enabled == false
```

Comment:
There will be conflict with my task

Reply:
i think this feature now will be under feature flag, we have to ask expert about this